frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tsx

```diff
@@ -96,29 +84,26 @@ const PoliciesTable = ({
     if (searchQuery) {
       delete emptyPolicies.graphicName;
       delete emptyPolicies.primaryButton;
-      emptyPolicies.header = "No policies match the current search criteria.";
-      emptyPolicies.info =
-        "Expecting to see policies? Try again in a few seconds as the system catches up.";
+      emptyPolicies.header = "No matching policies";
+      emptyPolicies.info = "No policies match the current filters.";
     }
 
     return emptyPolicies;
   };
 
   const searchable = !(policiesList?.length === 0 && searchQuery === "");
 
+  const hasPermissionAndPoliciesToDelete =
+    canAddOrDeletePolicy && hasPoliciesToDelete;
```

Comment:
Re: The inherited, it's not actually calling the API count that will hide that column, but the logic is in there in `hasPoliciesToDelete`